lua/neo-tree/sources/common/commands.lua

```diff
@@ -530,9 +530,19 @@ M.show_file_details = function(state)
     table.insert(left, "Size")
     table.insert(right, utils.human_size(stat.size))
     table.insert(left, "Created")
-    table.insert(right, os.date("%Y-%m-%d %I:%M %p", stat.birthtime.sec))
+    local created_format = require("neo-tree").config.default_component_configs.created.format
```

Comment:
hm, maybe it would be useful if this (and `modified_format`) were to be configurable in the mapping level as well? could be something like:

```suggestion
    local created_format = state.config.created_format or require("neo-tree").config.default_component_configs.created.format
```

Then the user could optionally specify:

```lua
mappings = {
  ['I'] = {
    'show_file_details',
    config = {
      created_format = '...',
      modified_format = '...'
    }
  }
}
```

so the details popup can time differently than the file tree.

It would require more documentation in `defaults.lua`, the vim help doc, and maybe the README though. LMK your thoughts on it.

Reply:
Sounds good!

I think the mappings config should be separate from the default component config to avoid confusion. Changes to the mappings config should only affect the file details popup, and changes to the component config should only affect the info column in the file explorer.

I'll work on an implementation and add in the necessary documentation soon!